darwin/datatypes.py

```diff
@@ -604,7 +701,155 @@ def make_cuboid(
     Annotation
         A cuboid ``Annotation``.
     """
-    return Annotation(AnnotationClass(class_name, "cuboid"), cuboid, subs or [], slot_names=slot_names)
+    return Annotation(AnnotationClass(class_name, "cuboid"), cuboid, subs or [], slot_names=slot_names or [])
+
+
+def make_table(
```

Comment:
I appreciate this is nice to have but maybe this could be in a separate PR? Doesn't feel necessary for the migration.

Reply:
It is required. We allow those types to be exported, and since our fallback to Darwin JSON 1.0 works by converting from 2.0 via darwin-py, darwin-py has to support **all** annotation types, otherwise we'll get regression on export.